akka-actor/src/main/scala/akka/io/UdpConnection.scala

```diff
@@ -120,7 +120,10 @@ private[io] class UdpConnection(
     }
     val buffer = bufferPool.acquire()
     try innerRead(BatchReceiveLimit, buffer)
-    finally {
+    catch {
+      case _: PortUnreachableException =>
+        if (TraceLogging) log.debug("Ignoring PortUnreachableException in doRead")
```

Comment:
would be kind of nice to log this at INFO or even WARN once, but indeed it would be excessive to send it for each read and adding logic/state here would probably not be worth it. so :+1:

Reply:
I think that it can be debug/trace as it is the expected behavior with UDP